src/components/PassthroughForm.vue

```diff
@@ -98,6 +98,12 @@ export default {
           let updatedForm = that.readFormFromResponse(response);
           let updatedHelpText = that.readHelpTextFromResponse(response);
           if (updatedForm.length == 1) {
+            // Login form can also redirect to change password form.
+            // If the input button says "Change password" redirect to the
+            // correct Vue route. Otherwise this form will post to the wrong endpoint.
+            if (updatedForm.find("input[value='Change password']").length == 1)
```

Comment:
I'm a little afraid this was meant to be a generic PassthroughForm (not just for Login potentially), so it might be misleading to have this specific redirect hardcoded into here. It might be better to add a slot into here and then just add this method to check and redirect in the Login specific instance of the PassthroughForm?

Reply:
There's tons of ways to do anything in vue, but maybe something like emitting an event in the success of that ajax call with the response, and then allow the parent LoginForm to change its property for the `successRedirectPath` based on the response. All other things using the PassthroughForm could ignore the success response event and behavior shouldn't change for them.

Reply:
Ok, updated to use events.